rero_mef/agents/cli.py

```
@@ -66,62 +121,98 @@ def get_pids_from_json(json_file):
+        app = current_app._get_current_object()
+
+        def _fetch_viaf(task):
+            mef_pid, viaf_source_code, entity_pid = task
+            with app.app_context():
+                try:
+                    data, msg = AgentViafRecord.get_online_record(
+                        viaf_source_code=viaf_source_code,
+                        pid=entity_pid,
+                    )
+                    return mef_pid, entity_pid, data, msg
+                except Exception as err:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)